msg/BoundingBox2D.msg

```diff
@@ -1,9 +1,9 @@
 # Note: This message plans to move to geometry_msgs in the final implementation.
 
-# The x/y position and (optional) rotation of the bounding box center.
-geometry_msgs/Pose2D pose
+# The 2D position (in pixels) and orientation of the bounding box origin.
+geometry_msgs/Pose2D origin
```

Comment:
Rotated 2D bounding boxes often is used for grasp generators and human annotations. Since the bounding boxes will probably move to geometry_msgs, I think that a generic rotated version should be best.